tests/test_check_cloudformation.py

```diff
@@ -0,0 +1,145 @@
+import os
+import shutil
+from unittest.mock import patch
+
+import pytest
+from click.testing import CliRunner
+from pathlib import Path
+
+from commands.check_cloudformation import check_cloudformation as check_cloudformation_command
+from commands.exceptions.CheckCloudformationFailure import CheckCloudformationFailure
+
+BASE_DIR = Path(__file__).parent.parent
```

Comment:
Is this the project root directory? I assume this test would fail if we move it elsewhere.

```suggestion
BASE_DIR = os.path.dirname(os.path.abspath(__file__))
```

Reply:
It would still fail if we moved things about with your suggestion. I'm not clear on the benefits of using one approach over another.

Reply:
You are right! We could have this in the `utils` file, set it there and then import it. Not a big deal though, of course.

Reply:
I have now realised that we already set this in the `conftest.py` file, so we could simply import it here.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/60/commits/a8569e306874b8c83f5c846cc7a9afc54ae47093.